lib/quickstart.rb

```diff
@@ -0,0 +1,5 @@
+require 'sinatra'
+
+get '/' do 
```

Comment:
Is that a wandering sinatra app in this project? You could probably delete it if it's not used :)